api/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+list(APPEND api_list
+    MWFunctions
+    MWOperators
+    Gaussians
+    Parallel
+    Plotter
+    Printer
+    Timer
+    )
+
+# Sets install directory for all the headers in the list                         
```

Comment:
The copy and install can be done more compactly without the loop:
```
file(
  COPY ${CMAKE_CURRENT_LIST_DIR}
   DESTINATION ${PROJECT_BINARY_DIR]/${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
 )
install(
   DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
  )
```
but please test that is works as expected!

Reply:
These commands work, but the api files now goes to `include/MRCPP/api/`. Is this how it has to be, or  can I circumvent the `api` layer in some way?

Reply:
If you try with the `${CMAKE_CURRENT_LIST_DIR}/*`? If that doesn't work disregard this comment altogether.